website/pages/community/index.jsx

```diff
@@ -1,10 +1,11 @@
 import VerticalTextBlockList from '@hashicorp/react-vertical-text-block-list'
 import SectionHeader from '@hashicorp/react-section-header'
 import Head from 'next/head'
+import s from './style.module.css'
 
 export default function CommunityPage() {
   return (
-    <div id="p-community">
+    <div className={s.root}>
```

Comment:
Removes the deprecated `--site-max-width` custom property, and refactors to CSS modules.